### ISSUE TITLE
Check print.control.net output for built-in 2-group models

### ISSUE THREAD

**Issue by smjenness** [statnet/EpiModel#320]
For example: 

```r
num1 <- num2 <- 250
nw <- network::network.initialize(num1+num2, directed = FALSE)
nw <- network::set.vertex.attribute(nw, "group", rep(c(1,2), each = num))
formation <- ~ edges + nodematch("group") + nodefactor("group")
target.stats <- c(150, 125, 62.5)
coef.diss <- dissolution_coefs(dissolution = ~ offset(edges), duration = 50)
est2 <- netest(nw, formation, target.stats, coef.diss)
dx2 <- netdx(est2, nsims = 5, nsteps = 250)
print(dx2)

#Parameters
init <- init.net(i.num = 5, i.num.g2 = 5)
param <- param.net(inf.prob = 0.1, inf.prob.g2 = 0.2,
                   rec.rate = 0.02, rec.rate.g2 = 0.02,
                   act.rate = 5)
control <- control.net(type = "SIS", nsteps = 100, nsims = 5)
control
```

provides this output:
```r
Network Model Control Settings
===============================
type = SIS
nsteps = 100
start = 1
nsims = 5
ncores = 1
rec.rand = TRUE
a.rand = TRUE
d.rand = TRUE
tea.status = TRUE
use.pids = TRUE
module.order =
save.nwstats = TRUE
nwstats.formula = formation
delete.nodes = FALSE
save.transmat = TRUE
save.network = TRUE
verbose = TRUE
verbose.int = 1
skip.check = FALSE
attr.rules = <list>
f.names = arrivals.FUN departures.FUN infection.FUN recovery.FUN get_prev.FUN
f.args = arrivals.net departures.net infection.net recovery.net get_prev.net
```

shouldn't this list the names of the 2-group functions under f.args?

**Comment by smjenness** [comment]
@andsv2 Reminder on this

**Comment by smjenness** [comment]
Actually, revert the last commit that updates crosscheck.net. I wasn't considering that print.control.net gets run before crosscheck.net. Instead, I would like print.control.net just to print out the module names in the order in which they are evaluated, and not print the function names.

So it should look like this, but with updates on the module names (removing the initialize and verbose ones) and ordering based on the Overhaul work:
```r
Network Model Control Settings
===============================
type = SIS
nsteps = 100
start = 1
nsims = 5
ncores = 1
rec.rand = TRUE
a.rand = TRUE
d.rand = TRUE
tea.status = TRUE
use.pids = TRUE
module.order =
save.nwstats = TRUE
nwstats.formula = formation
delete.nodes = FALSE
save.transmat = TRUE
save.network = TRUE
verbose = TRUE
verbose.int = 1
skip.check = FALSE
attr.rules = <list>
Dynamic Modules: departures.FUN arrivals.FUN recovery.FUN 
edges_correct.FUN resim_nets.FUN infection.FUN get_prev.FUN
```

**Comment by smjenness** [comment]
If the user specifies another order in module.order, this should take that into account.

**Comment by smjenness** [comment]
Closed with #377
